packages/server/src/users/users.controller.ts

```diff
@@ -1,7 +1,76 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Put, Query, UseGuards } from '@nestjs/common';
+import { ApiOperation, ApiSecurity, ApiTags } from '@nestjs/swagger';
+import { JWT_SECURITY } from '../auth/auth.constants';
+import { JwtAuthGuard } from '../auth/guards/jwt.guard';
+import { RolesGuard } from '../auth/guards/role.guard';
+import { CurrentUser } from '../common/decorators/current-user.decorator';
+import { Roles } from '../common/decorators/user-role.decorator';
+import { UpdateCriteriaDto } from './dto/update-criteria.dto';
+import { User, UserRole } from './entities/user.entity';
 import { UsersService } from './users.service';
 
 @Controller('users')
+@ApiTags('Users')
+@UseGuards(JwtAuthGuard, RolesGuard)
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
+
+  @Get('/criteria')
+  @ApiSecurity(JWT_SECURITY)
+  @ApiOperation({
+    summary: 'Get notification criteria for a user',
+    description:
+      'Get the critera (councils/meeting updates) that determines when a user will receive a notification',
+  })
+  async getUserNotificationCriteria(@CurrentUser() user: User): Promise<string[]> {
+    const usr = await this.usersService.getUser(user.email, true);
+    return usr.councils;
+  }
```

Comment:
Just return the entire User entity instead of returning a portion of it. We'll need the entire user for the UI anyway, two birds with one stone.

```suggestion
  @Get()
  @ApiSecurity(JWT_SECURITY)
  @ApiOperation({
    summary: 'Get a user',
    description: 'Returns the current user that is logged in.',
  })
  async getUser(@CurrentUser() user: User): Promise<string[]> {
    return this.usersService.getUser(user.email, true);
  }
```